source/camera/basic/capture_hdr_complete_settings.py

```diff
@@ -17,29 +17,38 @@ def _main():
     camera = app.connect_camera()
 
     print("Configuring the camera settings")
-    iris_setting = [17, 27, 27]
-    exposure_setting = [10000, 10000, 40000]
-    gain_setting = [1.0, 1.0, 2.0]
-    settings_collection = [camera.settings for _ in range(3)]
-    for i in range(len(settings_collection)):
-        settings_collection[i].iris = iris_setting[i]
-        settings_collection[i].exposure_time = datetime.timedelta(microseconds=exposure_setting[i])
-        settings_collection[i].brightness = 1
-        settings_collection[i].gain = gain_setting[i]
-        settings_collection[i].bidirectional = 0
-        settings_collection[i].filters.contrast.enabled = True
-        settings_collection[i].filters.Contrast.threshold = 0.5
-        settings_collection[i].filters.gaussian.enabled = True
-        settings_collection[i].filters.gaussian.sigma = 1.5
-        settings_collection[i].filters.outlier.enabled = True
-        settings_collection[i].filters.outlier.threshold = 5
-        settings_collection[i].filters.reflection.enabled = True
-        settings_collection[i].filters.saturated.enabled = True
-        settings_collection[i].blue_balance = 1
-        settings_collection[i].red_balance = 1
+
+    settings = zivid.Settings(
+        acquisitions=[
+            zivid.Settings.Acquisition(
+                aperture=8.0, exposure_time=datetime.timedelta(microseconds=10000), brightness=1.8, gain=1.0,
+            ),
+            zivid.Settings.Acquisition(
+                aperture=4.0, exposure_time=datetime.timedelta(microseconds=10000), brightness=1.8, gain=1.0,
+            ),
+            zivid.Settings.Acquisition(
+                aperture=4.0, exposure_time=datetime.timedelta(microseconds=40000), brightness=1.8, gain=2.0,
+            ),
+        ],
+    )
+    filters = settings.processing.filters
+    filters.noise.removal.enabled = True
+    filters.noise.removal.threshold = 10
+    filters.smoothing.gaussian.enabled = True
+    filters.smoothing.gaussian.sigma = 1.5
+    filters.outlier.enabled = True
+    filters.outlier.threshold = 5
+    filters.experimental.contrast_distortion.correction.enabled = True
+    filters.experimental.contrast_distortion.correction.strength = 0.4
+    filters.experimental.contrast_distortion.removal.enabled = False
+    filters.experimental.contrast_distortion.removal.threshold = 0.5
+    balance = settings.processing.color.balance
+    balance.red = 1.0
+    balance.blue = 1.0
+    balance.green = 1.0
```

Comment:
Maybe add a `print(settings)` after this? Nice opportunity to let the user know that they can print Settings to get a nicely formatted view of what they have.

Reply:
Also a good way of catching errors, which is easy to do since Python is so permissive with classes :)

Reply:
done, slightly differently.

Reply:
I think it would be safer to do:
```
for acquisition in settings.acquisitions:
    print(acquisition)
```
if not, any new acquisition added will not be printed.

However, why not just do `print(settings)` to get the whole thing?

Reply:
I will implement the first suggestion. Because I want to keep it as equal as possible with this sample for cpp and c# where we also have print outs separated. I think it is useful to show that this can be done. One can easily deduce how to print everything from this then.

Reply:
done

Reply:
Can't see it. Did you forget to push?

Reply:
pushed it to a wrong line of code. Check now.

Reply:
Looks good.